sdk/communication/Azure.Communication.Administration/README.md

```diff
@@ -116,6 +116,25 @@ Phone plans come in two types; Geographic and Toll-Free. Geographic phone plans
 
 All geographic phone plans within the same country are grouped into a phone plan group with a Geographic phone number type. All Toll-Free phone plans within the same country are grouped into a phone plan group.
 
+### Authenticate the Phone Number Administration client
+
+Phone Number Administration clients can be authenticated using connection string acquired from an Azure Communication Resources in the [Azure Portal][azure_portal].
+
+```C# Snippet:CreatePhoneNumberAdministrationClient
+// Get a connection string to our Azure Communication resource.
+var connectionString = "<connection_string>";
+var client = new PhoneNumberAdministrationClient(connectionString);
+```
+
+Phone Number Administration clients also have the option to authenticate using a valid token. With this option,
```

Comment:
nit. This is the terminology we have been using with the other platforms (more or less)
`Phone Number Administration clients also have the option to authenticate with Azure Active Directory Authentication`

Reply:
Thank you, I will update